internal/service/flexcluster/model_test.go

```diff
@@ -0,0 +1,53 @@
+//nolint:gocritic
+package flexcluster_test
+
+// import (
+// )
+
+// type sdkToTFModelTestCase struct {
```

Comment:
i think it would be better not to have commented-out code, maybe refer the commit in the PR description when they're removed, so anybody can go to that commit to see what commented code was there, in case you want to use in following PRs

Reply:
This commented out code will be removed/uncommented in the next PR that we do. This and the fact that it's in a dev branch is why I think it's not worth it to remove it for such a short amount of time